From: Justus Winter <justus@g10code.com>
Date: Mon, 19 Sep 2016 15:25:01 +0200
Subject: doc: Implement simple '@ref'erences.

* doc/yat2m.c (proc_texi_cmd): Handle '@ref'.

Signed-off-by: Justus Winter <justus@g10code.com>
(cherry picked from commit 55d560358f7ee4a8b68f98e051021a05d4c0bb1f)
---
 doc/yat2m.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/doc/yat2m.c b/doc/yat2m.c
index 9b76f19..a63260b 100644
--- a/doc/yat2m.c
+++ b/doc/yat2m.c
@@ -729,6 +729,7 @@ proc_texi_cmd (FILE *fp, const char *command, const char *rest, size_t len,
     { "asis",    7 },
     { "anchor",  7 },
     { "cartouche", 1 },
+    { "ref",     0, "[", "]" },
     { "xref",    0, "see: [", "]" },
     { "pxref",   0, "see: [", "]" },
     { "uref",    0, "(\\fB", "\\fR)" },
